packages/redux-dynamic-modules-observable/src/index.ts

```diff
@@ -1,12 +1,12 @@
-import { createEpicMiddleware } from "redux-observable";
 import { IExtension } from "redux-dynamic-modules-core";
 import { getEpicManager } from "./EpicManager";
+import { EpicMiddleware } from "redux-observable";
 import { IEpicModule } from "./Contracts";
 
 export * from './Contracts'
 
-export function getObservableExtension(): IExtension {
-    const epicMiddleware = createEpicMiddleware();
+export function getObservableExtension(_epicMiddleware: EpicMiddleware<any>): IExtension {
```

Comment:
`_epicMiddleware: EpicMiddleware<any> = createEpicMiddleware()`
For backward compatibility